client/my-sites/plugins/plugin-navbar-header/index.jsx

```diff
@@ -0,0 +1,662 @@
+											</svg>
+											<span className="x-hidden">{ translate( 'WordPress.com' ) }</span>
+										</a>
+									</li>
+									<li className="x-nav-item x-nav-item--wide">
+										<button
```

Comment:
Let's extract the repetitive markup for all the `button` and `li a` elements into helper React component. That could make the component smaller than it is now.

This PR is making the app entrypoints significantly larger, would be great if we could do something about it:
```
name         parsed_size           gzip_size
entry-main      +17670 B  (+1.3%)    +3278 B  (+0.9%)
entry-login     +17670 B  (+1.8%)    +3278 B  (+1.1%)
```